app/js/main.js

```diff
@@ -725,9 +766,11 @@ function _writeFile(file, data) {
     "use strict";
      fs.writeFile(file, data, function(err) {
         if (err) {
-            throw err;
+            console.error(`Failed to save ${file} to disk`);
+            checkDirectoryExists();
```

Comment:
This line goes back to my comment [here](https://github.com/BoatsAreRockable/animator/pull/94#issuecomment-165914811) but I think this will be fine to keep. I am concerned that you dropped the `err` object. Maybe you can `console.error(err);` that? It can provide vital information.